documentation/modules/rn-2.5.adoc

```diff
@@ -26,26 +26,36 @@ In this version of {project-short}, migration using OpenStack source providers g
 
 .Disabling FIPS
 
-EMS enforcement is disabled for migrations with VMware vSphere source providers to enable migrations from versions of vSphere that are supported by {project-short} but do not comply with the 2023 FIPS requirements.
+Enterprise Master Secret (EMS) enforcement is disabled for migrations with VMware vSphere source providers to enable migrations from versions of vSphere that are supported by {project-short} but do not comply with the 2023 FIPS requirements.
 
 .Integration of the create and update provider user interface
 
 The user interface of create and update providers now aligns with the look and feel of the {ocp} web console and displays up-to-date data.
 
 .Standalone UI
 
-The old UI of MTV 2.3 cannot be enabled by setting `feature_ui: true` in ForkliftController anymore.
+The old UI of {project-short} 2.3 cannot be enabled by setting `feature_ui: true` in ForkliftController anymore.
 
 .Errors logged in populator pods are improved
 
-In previous releases of {project-short} {project-version}, populator pods were always restarted on failure. This made it difficult to gather the logs from the failed pods. In {project-short} 2.5.3, the number of restarts of populator pods is limited to three times. On the third and final time, the populator pod remains in the fail status and its logs can then be easily gathered by must-gather and by forklift-controller to know this step has failed. link:https://issues.redhat.com/browse/MTV-818[(MTV-818)]
+In earlier releases of {project-short} {project-version}, populator pods were always restarted on failure. This made it difficult to gather the logs from the failed pods. In {project-short} 2.5.3, the number of restarts of populator pods is limited to three times. On the third and final time, the populator pod remains in the fail status and its logs can then be easily gathered by must-gather and by forklift-controller to know this step has failed. link:https://issues.redhat.com/browse/MTV-818[(MTV-818)]
+
+.`isPublic()` function executes code arbitrarily
+
+The Node.js Package Manager (npm) IP package has a vulnerability that allows an attacker to arbitrarily run code and obtain sensitive information. This vulnerability can allow a Server-Side Request Forgery (SSRF) attack, by using the `isPublic()` function to obtain access to resources that are normally protected.
+
+For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2265161[CVE-2023-42282]
+
+.Missing match property in Console 4.15
```

Comment:
that's true but it's too technical I think
I'd go with: "Support deployment on OpenShift 4.15" and then say that by fixing the operator failed on OpenShift 4.15 and the functionality in the UI broke due to that 'match' property that was dropped, and with fixes to these issues, as from 2.5.6 it is possible to deploy on OpenShift 4.15

Reply:
See updated text